src/v1/objects/insert.rs

```diff
@@ -429,4 +510,40 @@ impl super::Object {
 
         Ok(req_builder.method("POST").uri(uri).body(multipart)?)
     }
+
+    pub fn initiate_resumable_insert<'a, OID>(
```

Comment:
Naming consistency nit, `init_resumable_insert`? Since you are adding an `InitResumableInsertResponse`